map/place_page_info.hpp

```diff
@@ -100,7 +92,7 @@ class Info : public osm::MapObject
 
   // TODO(AlexZ): Temporary solution. It's better to use a wifi icon in UI instead of text.
   string m_localizedWifiString;
-  /// Booking rating string
-  string m_localizedRatingString;
+
```

Comment:
Ок
> On Jul 12, 2016, at 4:00 PM, Vlad Mihaylenko <notifications@github.com> wrote:
> 
> In map/place_page_info.hpp <https://github.com/mapsme/omim/pull/3767#discussion_r70432417>:
> 
> > @@ -100,7 +92,7 @@ class Info : public osm::MapObject
> >  
> >    // TODO(AlexZ): Temporary solution. It's better to use a wifi icon in UI instead of text.
> >    string m_localizedWifiString;
> > -  /// Booking rating string
> > -  string m_localizedRatingString;
> > +
> ну тут остальные поля это урлы, которые в поиске нам не нужны, а нужны только в плэйс пэйдже. m_isSponsoredHotel убрал.
> вообще, идея пул реквеста в том, чтобы данные для поисковой выдачи и данные для плэйс пэйджа ее отличались, чтобы в последствии можно было прокинуть например цену и рэйтинг букинга в поиск.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/mapsme/omim/pull/3767/files/001ad1c7e5a7fe4d637f5ad418848a1e9e890353#r70432417>, or mute the thread <https://github.com/notifications/unsubscribe/ACKTRJ78f1CzgM-iPnmPXaig1qcYnV-Aks5qU4_agaJpZM4JKV9D>.
>